packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/index.test.tsx

```diff
@@ -69,34 +83,15 @@ describe('PageAccountRecoveryConfirmKey', () => {
   //   bundle = await getFtlBundle('settings');
   // });
 
-  beforeEach(() => {
-    mockCode = 'code';
-    mockToken = 'token';
-    mockEmail = 'boo@boo.boo';
-    mockUid = 'uid';
-
-    account = {
-      getRecoveryKeyBundle: jest.fn().mockResolvedValue({
-        recoveryData: 'mockRecoveryData',
-        recoveryKeyId: MOCK_RECOVERY_KEY_ID,
-      }),
-      verifyPasswordForgotToken: jest
-        .fn()
-        .mockResolvedValue({ accountResetToken: MOCK_RESET_TOKEN }),
-    } as unknown as Account;
-  });
-
-  afterEach(() => {
-    jest.clearAllMocks();
-  });
-
-  it('renders as expected when the link is valid', () => {
-    render(<Subject {...{ account }} />);
+  it('renders as expected when the link is valid', async () => {
+    renderSubject(accountWithValidResetToken);
     // testAllL10n(screen, bundle);
 
-    const headingEl = screen.getByRole('heading', { level: 1 });
-    expect(headingEl).toHaveTextContent(
-      'Reset password with account recovery key to continue to account settings'
+    await waitFor(() =>
+      screen.findByRole('heading', {
```

Comment:
Can you use `await screen.findByRole` without the `waitFor`?

Reply:
Looks like yes! Thanks.